apps/platform-app/package.json

```diff
@@ -13,10 +13,13 @@
     "prisma:seed": "yarn prisma db seed",
     "postinstall": "prisma generate",
     "vercel-build": "prisma generate && prisma migrate deploy && next build",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "prisma:generate": "prisma generate --schema=$PRISMA_SCHEMA_PATH",
+    "prisma:studio": "prisma studio  --schema=./apps/platform-app/prisma/schema.prisma"
   },
   "prisma": {
-    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
+    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts",
+    "schema": "./apps/platform-app/prisma/schema.prisma"
```

Comment:
**nit:** We should remove this, as Vercel has troubles with deployment now

<img width="1330" alt="Zrzut ekranu 2024-06-11 o 13 24 26" src="https://github.com/nerdbord/good-dev-hunting-app/assets/22893188/a2e62a5a-cfeb-4bf0-a985-e2de7b93cbd3">